index.html

```diff
@@ -291,65 +454,198 @@ <h3>
+        The <a>TNF field</a> is `1` and the <a>TYPE field</a> is "`U`" (`0x55`).
+        The first byte of the <a>PAYLOAD field</a> is a URI identifier code,
+        in fact an index in an abbreviation table where the values are prepended
+        to the rest of the URI. For instance the value `0` denotes no prepending,
+        `1` denotes "`http://www.`", `0x04` denotes "`https://`"" and so on.
+        The rest of the payload contains the rest of the URI as a UTF-8 string
+        (and if the first byte is `0`, then it denotes the whole URI).
+      </p>
+      <p>
+        The URI is defined in [[RFC3987]] and in fact is a UTF-8 encoded IRI
+        that can be a URN or a URL.
+      </p>
+    </section>
+
+    <section> <h5>Smart poster record</h5>
     <p>
-      Other known RTD types supported by the platform and not Web NFC are
-      "`ac`" (`0x61`, `0x63`) for alternative carrier, "`Hc`" (`0x48`, `0x63`) for
-      handover carrier, "`Hr`" (`0x48`, `0x72`) for handover request, and "`Hs`"
-      (`0x48`, `0x73`) for handover select.
+      <dfn>Smart poster</dfn> is defined in [[NDEF-SMARTPOSTER]] as an NDEF
+      record that contains an <a>NDEF message</a> as payload, which may contain
+      several records: a mandatory <a>URI record</a> that refers to a content,
+      and additional optional records related to the content: a title record
+      (a <a>Text record</a>), one or more icon records, a type record, a size
+      record and an action record.
     </p>
     <p>
-      An <dfn>NFC handover</dfn> defines RTD and the
-      corresponding message structure that allows negotiation and activation of
-      an alternative communication carrier, such as Bluetooth or WiFi.
-      The negotiated communication carrier would then be used (separately) to
-      perform certain activities between the two devices, such as sending photos
-      to the other device, printing to a Bluetooth printer or streaming video to
-      a television set.
+      Icon records are <a>MIME type record</a>s. Readers SHOULD select one icon
+      to display.
     </p>
     <p>
-      Smart poster embeds additional records inside its payload. Below you
-      see an example embedding a text and a url record.
+      The type record has <a>NFC Forum local type</a> "`t`" specific to smart
+      poster and the <a>PAYLOAD field</a> contains a UTF-8 encoded MIME type for
+      the content referred to by the URI record.
+    </p>
+    <p>
+      The size record has <a>NFC Forum local type</a> "`s`" specific to smart
+      poster and the <a>PAYLOAD field</a> contains a 4-byte 32 bit unsigned
+      integer that denotes the size of the object referred to by the URL in the
+      URI record.
+    </p>
+    <p>
+      The action record  has <a>NFC Forum local type</a> "`act`" specific to
+      smart poster and the <a>PAYLOAD field</a> contains a single byte. If its
+      value is `0`, it means "do the action" (depending on type, this might mean
```

Comment:
maybe add a table for showing the values?

Reply:
Only 3 valid values... maybe later. Then I'd need to copy the table from the NDEF spec, something I didn't want to do.

Reply:
You can redo it from your own text

Reply:
That table would be incomplete compared to the NDEF spec, thus it would be confusing. Implementations are only interested in these 3 values. I can also remove this section (just say that it denotes an action specified in the NDEF spec), since Web NFC actually does not act on Sp, it leaves that to apps - so it's the app developers who should read the NDEF spec for Sp.

Reply:
No, please leave it

Reply:
OK - I am not even sure Sp records are not hijacked by the underlying platform. I think on Android they are not, but need to check.

Reply:
Made a table and added more verbose comments that didn't fit the table.